### ISSUE TITLE
Steam verification system

### ISSUE THREAD

**Issue by AkiraArtuhaxis** [AlexisL61/JackboxUtility#255]
The russian teams requested a verification system to be available in order to download the patches. This should be implemented somehow (depending on the Packs you have on Steam, you can download the patches from the games you have). Don't know what will be done with EGS tho.

**Comment by AlexisL61** [comment]
Small remark about it : every server will not have to use the verification system (as it could break EGS). It will be specific to some servers.

**Comment by 7aamusic** [comment]
They got warned from Jackbox Games because their files were distributed for free. On their site they added a Steam verification system that checks if there are licensed Jackbox Party Packs on your Steam account.

**Comment by AlexisL61** [comment]
Yes, and we will do something that will integrate seamlessly with their website, so they don't have a lot of work to do.
Here is how I see it : 

![image](https://github.com/AlexisL61/JackboxUtility/assets/30233189/a70e192d-06ea-413b-bb75-f7f04f67839a)

**Comment by AlexisL61** [comment]
There should be two verifications : 
- Local verification → Games added manually will not launch
- Online verification → Verification on a server
